### PR TITLE
Apply cleanup to changes introduced in pr 699

### PR DESCRIPTION
**Pull request by kprosise**
Applied fixes to grammar/style.

Ran linkcheck and linter, and viewed rendered HTML in browser.

No related ticket, quick fix

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)

## Overview

Applied changes suggested by @mike-scott, as well as a few other grammar/style changes.
Added some links to the glossary.

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for f953e0dc02f6907de2f3686e8a95943aa36e7652 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2783/docs/artifacts/html/index.html

File: source/reference-manual/security/device-gateway.rst [hunk 1]
```diff
@@ -3,11 +3,11 @@
 Managing Factory PKI
 ====================
 
-LmP devices connect to OTA services via a :ref:`Device Gateway <ref-ota-architecture>` configured with mutual TLS.
-A set of Digital Certificates used to establish trust between Factory devices and Device Gateway is a Factory Public Key Infrastructure (PKI).
+LmP devices connect to OTA services via a :ref:`Device Gateway <ref-ota-architecture>` configured with mutual :term:`TLS`.
```
Comment (vkhoroz) [inline source/reference-manual/security/device-gateway.rst:6]: Do we have a separate term for `mutual TLS`?

File: source/reference-manual/security/device-gateway.rst [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
+A set of digital certificates used to establish trust between Factory devices and a Device Gateway is a Factory Public Key Infrastructure (PKI).
```
Comment (vkhoroz) [inline source/reference-manual/security/device-gateway.rst:7]: Maybe, _**the** device gateway_? IIUC we only have the one of a kind.

**Review by vkhoroz — APPROVED** [review]
Thanks @kprosise!

LGTM

File: source/reference-manual/security/device-gateway.rst (hunk 1 above)
Comment (kprosise) [reply source/reference-manual/security/device-gateway.rst:6]: It appears we do, under mTLS. I will update it.

Comment (kprosise) [inline source/reference-manual/security/device-gateway.rst:6]:
```suggestion
LmP devices connect to OTA services via a :ref:`Device Gateway <ref-ota-architecture>` configured with :term:`mutal TLS <mTLS>`.
```

File: source/reference-manual/security/device-gateway.rst (hunk 2 above)
Comment (kprosise) [inline source/reference-manual/security/device-gateway.rst:7]:
```suggestion
A set of digital certificates used to establish trust between Factory devices and the Device Gateway is a Factory Public Key Infrastructure (PKI).
```

**Comment by doanac** [comment]
Docs for 8461038a6977fc48d053b6853a371191f65b7848 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2784/docs/artifacts/html/index.html

File: source/reference-manual/security/device-gateway.rst (hunk 1 above)
Comment (vkhoroz) [reply source/reference-manual/security/device-gateway.rst]: typo: mutal -> mutual.

Otherwise this looks good.

**Comment by doanac** [comment]
Docs for c5b2c08d3d77326eb0f34dec28e76599c5bd0904 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2786/docs/artifacts/html/index.html

**Pull request merged**